hamburguer.js

```diff
@@ -0,0 +1,21 @@
+const btnClose = document.querySelector('#btn-close');
+const divHamburguer = document.querySelector('.hamburguer');
+const iconHamburguer = document.querySelector('#hamburguer-icon');
+const hamburguerLink = document.querySelectorAll('.hamburguer-link');
+
+btnClose.addEventListener('click', () => {
+  divHamburguer.classList.add('run-out');
+  divHamburguer.classList.remove('run');
+});
+
+iconHamburguer.addEventListener('click', () => {
+  divHamburguer.classList.add('run');
+  divHamburguer.classList.remove('run-out');
+});
```

Comment:
- [ ] **[OPTIONAL]:** You did a great job writing JavaScript! 💯 However, at the moment, users can scroll over the page when the menu is open, which isn't a great user experience. If you would like to improve further this menu, I suggest you prevent the body from scrolling when it is open. You can achieve this by toggling a class that adds "overflow: hidden" to the body and removes it when hamburger/close buttons are clicked respectively. 🤓 🛠️ 

This is just an idea, feel free to implement your own solution! 🤓 

_--- JS ---_

```js
btnClose.addEventListener('click', () => {
  divHamburguer.classList.add('run-out');
  divHamburguer.classList.remove('run');
  
  body.classList.remove("no-scroll")
});

iconHamburguer.addEventListener('click', () => {
  divHamburguer.classList.add('run');
  divHamburguer.classList.remove('run-out');
  
  body.classList.add("no-scroll")
});
```

_-- CSS ---_

```CSS
.no-scroll {
  overflow: hidden;
}
```